src/services/SessionService.js

```diff
@@ -41,27 +43,34 @@ export default {
       sessionType,
       sessionSubTopic
     }).then(res => {
-      const data = res.data || {};
-      const { sessionId } = data;
-
-      this.currentSession.sessionId = sessionId;
+      return UserService.getUser(context).then(user => {
```

Comment:
@dmnisson What's the reason for fetching the user here?

I guess their `pastSessions` will have changed since starting the session, but is there anything else?

Reply:
This was just a leftover piece of code from before I changed the session logic to retrieve the user from the vuex store. I've removed this unnecessary fetch. Good catch.